static/logos/google_gcc.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
I think the filename needs to be changed.

Reply:
Good catch :D